rpcs3/Emu/Cell/lv2/sys_mutex.cpp

```diff
@@ -245,12 +245,38 @@ error_code sys_mutex_lock(ppu_thread& ppu, u32 mutex_id, u64 timeout)
 
 				std::lock_guard lock(mutex->mutex);
 
-				if (!mutex->unqueue(mutex->control.raw().sq, &ppu))
+				bool success = false;
+
+				mutex->control.fetch_op([&](lv2_mutex::control_data_t& data)
 				{
-					break;
+					success = false;
```

Comment:
obsolete?

Reply:
Wdym? fetch_op is a loop, it can execute more than once.